### PR TITLE
Add images, styling, and functionality

### PR DESCRIPTION
**Pull request by TahleeJ**
I added additional information to my "work experience" section about my time at STEP. 

I also added in JavaScript functionality: the "projects", "educational background", and "work experience" sections have a slideshow function that switches between different entries (with images that I added in as well).

I also edited the CSS to include a different font, and I also edited the backgrounds of my website. I also centered most of the elements on my website, including the text and images.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+for (i = 0; i < 3; i++) {
+    showSlides(slideIndices[i], i);
+}
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+function plusSlides(n, o) {
```
Comment (cshei) [inline portfolio/src/main/webapp/script.js:8]: Consider adding function comments to describe what each of these functions does -- it's a little unclear at first glance.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -1,28 +1,27 @@
...
+    showSlides(slideIndices[o] += n, o);
```
Comment (cshei) [inline portfolio/src/main/webapp/script.js:9]: I would pull the += out and do something like:

slideIndices[o] += n;
showSlides(slideIndices[o], o);

to make it more clear what the order of operations is and that the updated value is what's passed into the function call.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -1,28 +1,27 @@
...
+}
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function currentSlide(n) {
+    for (i = 0; i < 3; i++) {
+        showSlides(slideIndices[i] = n, i);
```
Comment (cshei) [inline portfolio/src/main/webapp/script.js:14]: ditto with the assignment

**Review by cshei — APPROVED** [review]
LGTM modulo minor comments

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (TahleeJ) [reply portfolio/src/main/webapp/script.js:8]: Ok, will do!

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (TahleeJ) [reply portfolio/src/main/webapp/script.js:14]: Oops, thank you for catching it

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (TahleeJ) [reply portfolio/src/main/webapp/script.js:9]: I'll add more clarity

**Pull request merged**